iocore/net/quic/QUICTokenCreator.h

```diff
@@ -0,0 +1,43 @@
+/** @file
+ *
+ *  A brief file description
+ *
+ *  @section license License
+ *
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+#pragma once
+
+#include "QUICFrameGenerator.h"
+#include "QUICContext.h"
+
+class QUICTokenCreator final : public QUICFrameGenerator
```

Comment:
Why do you want to mark this `final`?

Reply:
Because I don't want other derived from this object. It is final object.

Reply:
I understand what final means :-) Why does it have to be a final object?

Reply:
Because currently we don't have any class wants to derive from it .

Reply:
True, but there are many classes we wouldn't extend. It made me curious why this has `final`, and I thought it might have a special reason.